### DISCUSSION TITLE
Microsoft Teams Integration

### DISCUSSION THREAD

**Discussion by avivapinchas** [ParabolInc/parabol#5457] [New features]
For teams that communicate via Teams, we could offer a notification integration, so teams are informed about their meetings in the context of their existing comms tools. We've had ~10 requests for this. 

User requests: #3538

**Comment by enriquesanchez** [comment]
MS Teams was one of the top most requested integrations from the [Parabol Customer Feedback Survey](https://parabol.slack.com/archives/C08FL5UE7/p1633422567280700) 🔒.
